### PR TITLE
Updated Clinvar howto

### LINKED ISSUE #3730: Update ClinVar submission howto
This one:

image

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3730

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Go on the ClinVar submissionS page and click on the howto:

image

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 1]
```diff
@@ -26,37 +26,40 @@ <h3 class="mt-3">ClinVar submissions page<h5></h3><a data-bs-target="#howto" hre
...
-          </p><br>
-          <h5>Fill in panel 1 and 2</h5>
-          <p>
+                          If your Organization is not already present in ClinVar you should follow the instructions to <strong>register a submitting Organization</strong> to the same portal.
+                          The first time an organization is registered in the ClinVar Submission Portal, the content will be reviewed, so it will take some time before getting approved in order to start a variant submission.
+            <br><br>
+            <h5>Automatic submission</h4>
+            Once the requirements above are fullfilled you could submit using the API. Submission by API requires a <strong>service account key</strong> for the submitting organization and an API key associated with it.
+            Follow the isntructions provided on the <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/" target="_blank" rel="noopener">API pages</a> to obtain a valid key for the service account.<br>
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar_submissions.html:36]:
```suggestion
            Follow the instructions provided on the <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/" target="_blank" rel="noopener">API pages</a> to obtain a valid key for the service account.<br>
```

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 2]
```diff
@@ -26,37 +26,40 @@ <h3 class="mt-3">ClinVar submissions page<h5></h3><a data-bs-target="#howto" hre
...
-          </p><br>
-          <h5>Fill in panel 1 and 2</h5>
-          <p>
+                          If your Organization is not already present in ClinVar you should follow the instructions to <strong>register a submitting Organization</strong> to the same portal.
+                          The first time an organization is registered in the ClinVar Submission Portal, the content will be reviewed, so it will take some time before getting approved in order to start a variant submission.
+            <br><br>
+            <h5>Automatic submission</h4>
+            Once the requirements above are fullfilled you could submit using the API. Submission by API requires a <strong>service account key</strong> for the submitting organization and an API key associated with it.
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar_submissions.html:35]:
```suggestion
            Once the requirements above are fulfilled you could submit manually using files uploaded to the ClinVar submission portal. See the heading below. You can avoid this extra file handling step by submitting directly via API instead. Submission by API requires a <strong>service account key</strong> for the submitting organization and an API key associated with it.
```

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 3]
```diff
@@ -26,37 +26,40 @@ <h3 class="mt-3">ClinVar submissions page<h5></h3><a data-bs-target="#howto" hre
...
+            Follow the isntructions provided on the <a href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/" target="_blank" rel="noopener">API pages</a> to obtain a valid key for the service account.<br>
+            Once you are satisfied with the content of your submission, you could validate it and obtain a submission ID the ClinVar API by clicking on the <strong>Validate and obtain ID from ClinVar</strong> button:<br><br>
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar_submissions.html:37]:
```suggestion
            Once you are satisfied with the content of your submission, you could validate it and obtain a submission ID from the ClinVar API by clicking on the <strong>Validate and obtain ID from ClinVar</strong> button as in the image below. This does not require an institute specific API key.<br><br>
```

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 4]
```diff
@@ -26,37 +26,40 @@ <h3 class="mt-3">ClinVar submissions page<h5></h3><a data-bs-target="#howto" hre
...
-          </p><br>
-          <h5>Upload the files to ClinVar</h5>
-          <p>
-            Upload the files obtained above (either the variants .csv file or the variants .csv + the casedata .csv files) in the "<strong>Submission Files</strong>" field in the third panel of the submission.
-          The field named <strong>"Assertion Criteria File" should be left blank</strong>, since the default information is already filled in the "Variants sheet" <strong>(ACMG Guidelines, 2015)</strong>.<br>
-          <text class="text-danger"> Note that if you edit these files in Excel you need to save them in the correct format: <strong>comma-separated columns</strong>. Default settings should probably work fine if your software's locale is US English.
-          If this is not the case you might need to change your software's region settings to use US English before exporting these tables.</text>
-          </p>
+            <strong>Assemby name</strong> fields should be filled in before proceding further with the submission.<br>
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar_submissions.html:50]:
```suggestion
            <strong>Assemby name</strong> fields should be filled in before proceeding further with the submission.<br>
```

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 5]
```diff
@@ -26,37 +26,40 @@ <h3 class="mt-3">ClinVar submissions page<h5></h3><a data-bs-target="#howto" hre
...
+
+            <img width="202" alt="image" src="https://user-images.githubusercontent.com/28093618/208666249-2bfe457d-a194-4d5f-a856-97e75da14681.png"><br><br>
+
+            If the validation is successful, a submission name will be saved for your submission. Alternatively validation errors will be displayed on the page.
+            To submit your variants using the ClinVar API, you'll need to click on the <strong>Submit to ClinVar</strong> button:<br><br>
+
+            <img width="264" alt="image" src="https://user-images.githubusercontent.com/28093618/208666318-0efac91e-c5d2-42a0-9ceb-d7fc68c6d920.png"><br><br>
+
+            <h5>Manual submission</h5>
+            Once you have registered to the ClinVar portal as a submitter for a given Organization, you can start a new submission by using the "<strong>Upload new file submission</strong>" button from the <a href="https://submit.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener">submission page</a>.<br>
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar_submissions.html:47]:
```suggestion
            Once you have registered to the ClinVar portal as a submitter for a given Organization, you can start a new manual submission by using the "<strong>Upload new file submission</strong>" button from the <a href="https://submit.ncbi.nlm.nih.gov/clinvar/" target="_blank" rel="noopener">submission page</a>.<br>
```

**Review by dnil — APPROVED** [review]
Great! Some corrections suggestions here.

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html (hunk 3 above)
Comment (northwestwitch) [reply scout/server/blueprints/clinvar/templates/clinvar_submissions.html:37]: Right, nice suggestion!

**Comment by northwestwitch** [comment]
> Great! Some corrections suggestions here.

Much appreciated, thanks! Merging!

**Pull request merged**